src/bigquery.ts

```diff
@@ -2200,25 +2191,25 @@ export class BigQuery extends Service {
     if (!queryReq) {
       this.createQueryJob(query, (err, job, resp) => {
         if (err) {
-          (callback as SimpleQueryRowsCallback)(err, null, resp);
+          callback && callback(err, null, null, resp);
           return;
         }
         if (typeof query === 'object' && query.dryRun) {
-          (callback as SimpleQueryRowsCallback)(null, [], resp);
+          callback && callback(null, [], null, resp);
           return;
         }
         // The Job is important for the `queryAsStream_` method, so a new query
         // isn't created each time results are polled for.
         options = extend({job}, queryOpts, options);
-        job!.getQueryResults(options, callback as QueryRowsCallback);
+        job!.getQueryResults(options, callback);
```

Comment:
we don't typecast here anymore because there's only one option for callback type, right?